baybe/kernels/basic.py

```diff
@@ -0,0 +1,46 @@
+"""Collection of kernels."""
+
+from fractions import Fraction
+from typing import Union
+
+from attrs import define, field
+from attrs.validators import in_
+
+from baybe.kernels.base import Kernel
+
+
+def _convert_fraction(value: Union[str, float, Fraction], /) -> float:
+    """Convert the provided value into a float.
+
+    Args:
+        value: The parameter that should be converted.
+
+    Returns:
+        The converted parameter.
+
+    Raises:
+        ValueError: If `value` was provided as string but could not be interpreted as
+            fraction.
+    """
+    if isinstance(value, str):
+        try:
+            value = Fraction(value)
+        except ValueError as err:
+            raise ValueError(
+                f"Provided value of {value} of `value` could not be interpreted as "
+                f"fraction"
```

Comment:
```suggestion
                f"The provided input '{value}' could not be interpreted as a fraction."
```
(reason: see comment above)